doc/releases/release-notes-2.4.rst

```diff
@@ -599,7 +599,14 @@ Bluetooth
 Build and Infrastructure
 ************************
 
-* Devicetree
+* Devicetree:
+
+  * Added new devicetree macros that provide a default value if the property
+    or cell accessor doesn't contain the requested data
+
+  * Added support for inferring bindings for ``/zephyr,user`` devicetree node
+    to allow applications easy way to specify application specific devicetree
```

Comment:
something a bit strange with the syntax in this line

Reply:
what do you mean by syntax?

Reply:
maybe "to allow applications *an* easy way to..."

Reply:
Done